packages/mobile/src/recipients/RecipientItem.tsx

```diff
@@ -39,9 +39,7 @@ class RecipientItem extends React.PureComponent<Props> {
             <Text numberOfLines={1} ellipsizeMode={'tail'} style={styles.name}>
               {getDisplayName(recipient, t)}
             </Text>
-            {!recipient.name ? (
```

Comment:
This was my mistake, it should be `{recipient.name ? (`, but I don't think we should get rid of the conditional statement, or else if a recipient doesn't have a name, it will display their phone number/account address twice